internal/pkg/source/testdata/src/sanitization/test.go

```diff
@@ -22,7 +22,7 @@ type foo struct {
 }
 
 func f1() {
-	f := &foo{name: "n", password: "p"} // want "sanitizer"
+	f := &foo{name: "n", password: "p"} // want ""
```

Comment:
Ah, good point. I often forget that those are `regexp`.

My intent was actually to match nothing (so I should've used "^$"), since because the call to `sanitizer` precedes the call to `fmt.Println`, I was expecting (rather naively) that it wouldn't be reached.

This would be a valid expectation if the `sanitizer` returned a sanitized `Source`, but it modifies the `Source` that is passed to it as a pointer.

So it seems to me the real issue here is that we need a similar test in `pkg/levee` to prevent well-meaning contributors from removing useful code. :thinking: 

I'm going to close this PR and open a new one adding the missing test.